suite/tests/linux/syscall_pwait.cpp

```diff
@@ -197,7 +197,8 @@ main(int argc, char *argv[])
     auto psyscall_epoll_pwait = [test_set](bool nullsigmask) -> int {
         int epoll_fd = epoll_create1(EPOLL_CLOEXEC);
         struct epoll_event events;
-        return epoll_pwait(epoll_fd, &events, 24, -1, nullsigmask ? NULL : &test_set);
+        return epoll_pwait(epoll_fd, &events, IF_X64_ELSE(24LL, 24L),
+                           IF_X64_ELSE(-1LL, -1L), nullsigmask ? NULL : &test_set);
```

Comment:
This also looks incorrect: this is an int, not a long and not a long long.

Reply:
Same as above.